pytest_xmlcomp/plugin.py

```diff
@@ -100,14 +100,14 @@ def collect(self):
             print("JSON Syntax Error in file %s:\n%s" % (jsonfile, error),
                   file=sys.stderr)
             return
-        for xpath, expresult in jsondata:
-            yield XPathItem(xpath, self, expresult, tree)
+        namesp = None if not jsondata.get('ns') else dict(jsondata.get('ns'))
+        for xpath, expresult in jsondata['data']:
```

Comment:
Should we make a consistency check before accessing `jsongdata['data']`? Just in case the user doesn't add a `data` section.

Maybe add a separate test with a JSON file without a `data` section.